deploy/kubernetes/base_windows/controller.yaml

```diff
@@ -0,0 +1,82 @@
+kind: StatefulSet
```

Comment:
Since almost all the specs are the same except for the paths and container images, can we use the normal "base" specs, and just have a windows overlay that patches the paths and images?

That way we have one less place to make changes to the spec.

Reply:
I was thinking about this option. My concern was when we make some new changes that only apply to Linux, not Windows, we might break Windows installation. Now checking the "base" part, it seems not related to windows, so maybe there is no concern. What do you think?

Reply:
Agree, I'm not sure I see anything very specific to Linux or Windows besides the paths and images.